frontend/src/main/components/Nav/AppNavbar.js

```diff
@@ -73,6 +73,13 @@ export default function AppNavbar({ currentUser, systemInfo, doLogout, currentUr
                   </NavDropdown>
                 )
               }
+              {
+                hasRole(currentUser, "ROLE_ADMIN") && (
```

Comment:
```suggestion
                hasRole(currentUser, "ROLE_DRIVER") && (
```

Restricting this to ROLE_ADMIN sort of misses the point.  This is intended to be a page for the Drivers to see information that is relevant to their role.